modules/weather.md

```diff
@@ -89,7 +89,7 @@ The following properties can be configured:
 | `colored`         | If set to `true`, the min and max temperature are color coded. <br><br> **Default value:** `false`                                                                                                                                                                                                                                                                                                                                                                                                 |
 | `fade`            | Fade the future events to black. (Gradient) <br><br> **Possible values:** `true` or `false` <br> **Default value:** `true`                                                                                                                                                                                                                                                                                                                                                                         |
 | `fadePoint`       | Where to start fade? <br><br> **Possible values:** `0` (top of the list) - `1` (bottom of list) <br> **Default value:** `0.25`                                                                                                                                                                                                                                                                                                                                                                     |
-| `maxNumberOfDays` | How many days of forecast to return. Specified by config.js <br><br> **Possible values:** `1` - `16` <br> **Default value:** `5` (5 days) <br> This value is optional. By default the weather module will return 5 days.                                                                                                                                                                                                                                                                           |
+| `maxNumberOfDays` | How many days of forecast to return. Specified by config.js <br><br> **Possible values:** `1` - `16` <br> **Default value:** `5` (5 days) <br> This value is optional. By default the weather module will return 5 days. <br><br> Note: If using a free api key the max days you can retrieve is 5, so placing anything higher than 5 in this property will still return the default of 5. To retrieve more than 5 days you must subscribe to the OpenWeatherMap API.                                                                                                                                                                                                                                                                        |
```

Comment:
I would rather put the Note under the Openweathermap options (see EnvCanda note for details) since not all providers might have that limitation

Reply:
Oh yes sorry i thought i was in that section. let me update

Reply:
Updated to correct section.